lib/ncr/mailboxes.rb

```diff
@@ -0,0 +1,29 @@
+module Ncr
+  module Mailboxes
+    def self.ba61_tier1_budget
+      self.approver_with_role("BA61_tier1_budget_approver")
+    end
+
+    def self.ba61_tier2_budget
+      self.approver_with_role("BA61_tier2_budget_approver")
+    end
+
+    def self.ba80_budget
+      self.approver_with_role("BA80_budget_approver")
+    end
+
+    def self.ool_ba80_budget
+      self.approver_with_role("OOL_BA80_budget_approver")
+    end
+
+    protected
+
+    def self.approver_with_role(role_name)
```

Comment:
> Interesting that you see this module as a singleton...I usually think of modules in ruby as a set of behaviors to be mixed in multiple other classes

They can be either!

http://dalibornasevic.com/posts/9-ruby-singleton-pattern

 Thanks Ruby :unamused: